packages/ui/src/view/v2/GridCard/GridCardConnectPlaceholder.tsx

```diff
@@ -1,39 +1,44 @@
-import React from 'react';
-import { ApiOutlined, SafetyOutlined } from '@ant-design/icons';
+import React, { ReactNode } from 'react';
+import { ApiOutlined } from '@ant-design/icons';
 import { Button, ButtonProps, Space, Typography } from 'antd';
 
 import styles from './GridCardConnectPlaceholder.module.scss';
 
+export const DEFAULT_CARD_CONNECT_PLADHOLDER_DICTIONARY: TCardConnectPlaceholderDictionary = {
```

Comment:
Ca ne serait pas placeholder plutot ?
```suggestion
export const DEFAULT_CARD_CONNECT_PLACEHOLDER_DICTIONARY: TCardConnectPlaceholderDictionary = {
```